src/hazelcore/Operators_Typ.re

```diff
@@ -15,3 +15,14 @@ let to_parse_string = op =>
   | Arrow => "->"
   | _ => to_string(op)
   };
+
+let precedence =
+  fun
+  | Arrow => 0
+  | Sum => 1
+  | Prod => 2;
```

Comment:
Let's use the same precedence as specified in `HTyp`. We can also refactor any use of precedence in `HTyp` and `HTypDoc` to refer back to this as single source of truth.

Reply:
There's also some code in `UHTyp.contract` that should use the `precedence` function defined here.

Reply:
One last place where we should redirect precedence lookup: the `DHDoc_*` modules. Those modules should also call `Operators_*.precedence`.

Reply:
> Let's use the same precedence as specified in `HTyp`. We can also refactor any use of precedence in `HTyp` and `HTypDoc` to refer back to this as single source of truth.

Should I just rewrite the precedence lookup function in `HTyp` to reference `Operators_Typ`, and leave all the references to `HTyp.precedence*` alone?

Or should I remove all precedence lookup functions and change all references to be to the functions `Operator_*.precedence`?

Reply:
The former is fine